spread.yaml

```diff
@@ -1255,6 +1259,11 @@ suites:
 
             # Configure the ssh connection to the test vm
             remote.setup config --host localhost --port 8022 --user user1 --pass ubuntu
+
+            # Build and install initramfs package
+            if nested_is_core_ge 24; then
```

Comment:
I'm missing some context from the previous PRs, just to be sure, the intent is to start building initrd from snapd source tree for UC24+?

Reply:
Yes, that is correct, we have now core-initrd in snapd repo but only for UC24+-